jest/resolve-modules.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const VIRTUAL_MODULES = [
-    '@testing-library/react-native/cleanup-after-each',
+    // Add here modules that are not installed and for which you'd like, e.g., mock require.resolve
```

Comment:
As discussed, can you please change this message to something more clear?

```suggestion
    // Add here modules that are not installed but which you still need to require by using require.resolve
```